llvm/include/llvm/IR/VectorBuilder.h

```diff
@@ -57,6 +58,11 @@ class VectorBuilder {
     return RetType();
   }
 
+  // Helper function for creating VP intrinsic call.
```

Comment:
independent of this change, but if `VectorBuilder` only supports generation of vector-predication intrinsics, then it would be better to call it `VectorPredicationBuilder` to avoid confusion

Reply:
Indeed, this can be adjusted later. (But I would prefer a shorter name, perhaps just VectorPredBuilder would be good enough?)

Reply:
or `VPBuilder`, like there's `IRBuilder`

Reply:
I think it's worth being more explicit for the name in the utility in `llvm/IR`, `VectorPredBuilder` would sound good to me.

Reply:
nit: use `///` for doc-comment